src/libcsg/modules/io/groreader.cc

```diff
@@ -116,8 +119,22 @@ bool GROReader::NextFrame(Topology &top) {
       }
 
       // res -1 as internal number starts with 0
-      b = top.CreateBead<Bead>(1, atName, atName, residue_number - 1, resName,
-                               resName, 1., 0.);
+      byte_t symmetry = 1;
```

Comment:
why this is all needed?

Reply:
I am not a fan of hidden default variables. If a developer is forced to explicitly describe each attribute they will pay more attention to what they are doing. Though there is probably a better way to pass these values in as opposed to having a long list of parameters.